docker/webapp/php.ini

```diff
@@ -2,4 +2,4 @@ error_reporting=E_ALL & ~E_DEPRECATED
 
 [xdebug]
 xdebug.client_host=host.docker.internal
```

Comment:
host.docker.internal ironically doesn't work on Linux, it's strictly a Docker Desktop thing.  It should be this instead:

```
xdebug.client_host=172.17.0.1
```

That's what host.docker.internal resolves to in any default docker setup.  Anyone who has it set up differently is savvy enough to override it themselves.